README.md

```diff
@@ -27,8 +27,35 @@ A single network interface can be associated with the device.
   - Pre-allocated buffers for packet processing in static memory
 
 
+## Functional tests with `LD_PRELOAD`
+
+The POSIX shim builds `libwolfip.so`, which can be injected in front of
+host tools so that calls to `socket(2)` and friends are redirected to the
+wolfIP stack and the TAP device (`wtcp0`).  After running `make`:
+
+```sh
+sudo LD_PRELOAD=$PWD/libwolfip.so nc 10.10.10.2 80
```

Comment:
The ping example works well, but having trouble getting nc or netcat example to work with this.